src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -980,6 +996,17 @@ public void checkOptionsValid() {
       }
       classnames.add(classname);
     }
+
+    // This does not exclude explicitly-specified methods.  In other words, if the user specified a
+    // method explicitly, this does not exclude it even if it is in an excluded class.
+    for (Iterator<String> itor = classnames.iterator(); itor.hasNext(); ) {
+      String classname = itor.next();
+      if (shouldOmitClass(classname)) {
+        itor.remove();
+        break;
+      }
```

Comment:
I don't understand why there's the `break;` ? why don't you continue looping for the rest of the classnames?

Reply:
You are right.  That was left over from logic before `shouldOmitClass` was refactored into its own method.  Thanks for noticing!  I appreciate your review.